ONSdigital/rm-sample-service
# Fail build for bad code style

# Motivation and Context
The install phase of travis will fix any bad styling before the check so
this will never flag up in travis. This change moves the style check to
before the install so styling issues should be caught.

# What has changed
Check styling before it's fixed

# How to test?
1. Break the styling
1. Run the maven commands in the same order as travis will
1. Observe style error before it's fixed
